logstash-core/src/main/java/org/logstash/plugins/filters/Uuid.java

```diff
@@ -73,6 +73,12 @@ public Collection<Event> filter(Collection<Event> events, final FilterMatchListe
         return events;
     }
 
+    @Override
+    public void close() {
+        // no-op operation for UUID generator filter
+        return;
```

Comment:
With the above-mentioned default method, this shouldn't be needed. But it does provide us a good place _to test_ an implementation. Perhaps we could make this method make a change to the filter instance that we can validate after it has been closed?

Reply:
We shouldn't pollute production code with code useful just for testing.
I thought to create a fake filter in test, with the `close` method redefined, but the test ends in something that's trivial.
The `FakeFilter` redefines the `close()` method which suppose set a boolean flag to mark the execution. Then the testing code, instantiate the `FakeFilter`, invokes the `close` and verify the side effect; this means testing that calling a method the method is effectively called, which doesn't provide such value.

A more useful test would be to use the fake filter in a running pipeline, stop the pipeline and check that the filter is stopped. WDYT?

Reply:
🤦🏼 I failed to see that this is in `src/*main*/`. Absolutely.